server/sonar-web/src/main/js/apps/projects-admin/search.js

```diff
@@ -105,11 +105,11 @@ export default React.createClass({
         />
       </td>
     );
-  },
+  };
 
-  renderSpinner() {
+  renderSpinner = () => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)